pydantic/_internal/_decorators.py

```diff
@@ -636,43 +637,6 @@ def _wrapper2(
 ]
 
 
-_VALID_SERIALIZER_SIGNATURES = """\
-Valid serializer signatures are:
-
-# an instance method with the default mode or `mode='plain'`
-@serializer('x')  # or @serialize('x', mode='plain')
-def ser_x(self, value: Any, info: pydantic.FieldSerializationInfo): ...
-
-# a static method or free-standing function with the default mode or `mode='plain'`
-@serializer('x')  # or @serialize('x', mode='plain')
-@staticmethod
-def ser_x(value: Any, info: pydantic.FieldSerializationInfo): ...
-# equivalent to
-def ser_x(value: Any, info: pydantic.FieldSerializationInfo): ...
-serializer('x')(ser_x)
-
-# an instance method with `mode='wrap'`
-@serializer('x', mode='wrap')
-def ser_x(self, value: Any, nxt: pydantic.SerializerFunctionWrapHandler, info: pydantic.FieldSerializationInfo): ...
-
-# a static method or free-standing function with `mode='wrap'`
-@serializer('x', mode='wrap')
-@staticmethod
-def ser_x(value: Any, nxt: pydantic.SerializerFunctionWrapHandler, info: pydantic.FieldSerializationInfo): ...
-# equivalent to
-def ser_x(value: Any, nxt: pydantic.SerializerFunctionWrapHandler, info: pydantic.FieldSerializationInfo): ...
-serializer('x')(ser_x)
-
-For all of these, you can also choose to omit the `info` argument, for example:
-
-@serializer('x')
-def ser_x(self, value: Any): ...
-
-@serializer('x', mode='wrap')
-def ser_x(self, value: Any, handler: pydantic.SerializerFunctionWrapHandler): ...
-"""
```

Comment:
Probably a good idea to store this (or a reference to it anyway) somewhere for inclusion in the 'field-serializer-signature' errors page (I guess with some modifications).

Reply:
Nevermind I see how it works, and you already did it.